CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [unreleased]
 
+## [0.0.45] - 2024-04-25
+
+-   Adds `key={Date.now()}` in the `TryRefreshComponent` to fix the issue with the `TryRefreshComponent` not running `useEffect` callback if session is refreshed multiple times.
+
```

Comment:
no need to make this since it's only the example app changes. A version bump is only necessary if we are making CLI code changes